config/initializers/captchas.rb

```diff
@@ -2,9 +2,12 @@
 
 Recaptcha.configure do |config|
   # Do not verify recaptcha keys if it is not correctly configured
-  (config.skip_verify_env ||= []) << Rails.env if Rails.configuration.three_scale.recaptcha_private_key.blank?
+  (config.skip_verify_env ||= []) << Rails.env if Rails.configuration.three_scale.recaptcha_public_key.blank?
```

Comment:
`recaptcha_public_key` is the only parameter which is mandatory for both classic and enterprise, so it's the best way to know if bot protection should be enabled.